llvm/test/CodeGen/Mips/nofpclass.ll

```diff
@@ -16,3 +16,45 @@ entry:
   %cond = tail call float @llvm.maximumnum.f32(float %a, float %b)
   ret float %cond
 }
+
+define { float, float } @struct({ float, float} nofpclass(nan) %a) {
+; MIPS32R6-LABEL: struct:
+; MIPS32R6:       # %bb.0:
+; MIPS32R6-NEXT:    mov.s $f2, $f14
+; MIPS32R6-NEXT:    jr $ra
+; MIPS32R6-NEXT:    mov.s $f0, $f12
+;
+; MIPS64R6-LABEL: struct:
+; MIPS64R6:       # %bb.0:
+; MIPS64R6-NEXT:    mov.s $f2, $f13
+; MIPS64R6-NEXT:    jr $ra
+; MIPS64R6-NEXT:    mov.s $f0, $f12
```

Comment:
Missing a use that shows this did something

Reply:
Let's remove them since we have some new tests with max.